source/operations/concepts/architecture.rst

```diff
@@ -0,0 +1,189 @@
+
+   .. figure:: /images/architecture/architecture-multi-site.svg
+      :figwidth: 100%
+      :alt: Diagram of a multi-site deployment with three MinIO peer site
+
+      A MinIO multi-site deployment with three peers.
+      Write operations on one peer replicate to all other peers in the configuration automatically.
+
+Each peer site consists of an independent set of MinIO hosts, ideally having matching pool configurations.
+   The architecture of each peer site should closely match to ensure consistent performance and behavior between sites.
+   All peer sites must use the same primary identity provider, and during initial configuration only one peer site can have any data.
+
+   .. figure:: /images/architecture/architecture-multi-site-setup.svg
+      :figwidth: 100%
+      :alt: Diagram of a multi-site deployment during initial setup
+
+      The initial setup of a MinIO multi-site deployment.
+      The first peer site replicates all required information to other peers in the configuration.
+      Adding new peers uses the same sequence for synchronizing data.
+
+Replication performance primarily depends on the network latency between each peer site.
+   With geographically distributed peer sites, high latency between sites can result in significant replication lag.
+   This can compound with workloads that are near or at the deployment's overall performance capacity, as the replication process itself requires 
+
+   .. figure:: /images/architecture/architecture-multi-site-latency.svg
+      :figwidth: 100%
+      :alt: Diagram of a multi-site deployment with latency between sites
+
+      In this peer configuration, the latency between Site A and it's peer sites is 100ms.
+      The earliest point in time in which the object fully synchronizes to all sites is at least 110ms.
+
+Deploying a global load balancer or similar network appliance with support for site-to-site failover protocols is critical to the functionality of multi-site deployments.
+   The load balancer should support a health probe/check setting to detect the failure of one site and automatically redirect applications to any remaining healthy peer.
+
+   .. figure:: /images/architecture/architecture-load-balancer-multi-site.svg
+      :figwidth: 100%
+      :alt: Diagram of a multi-site deployment with a failed site
+
+      One of the peer sites has failed completely.
+      The load balancer automatically routes requests to the remaining healthy peer site.
+
+   The load balancer should meet the same requirements as single-site deployments regarding connection balancing and header preservation.
+   Once all data synchronizes, you can restore normal connectivity to that site.
+   Depending on the amount of replication lag, latency between sites, and overall workload IO, you may need to temporarily stop write operations to allow the sites to completely catch up.tions to other sites in the deployment.
```

Comment:
Seems to be some kind of copy/paste or delete error in this sentence.

> catch up to other sites in the deployment.

Maybe?